src/experimental-layers/src/mesh-layer/mesh-layer.js

```diff
@@ -108,6 +125,14 @@ export default class MeshLayer extends Layer {
       },
       instanceColors: {size: 4, accessor: 'getColor', update: this.calculateInstanceColors}
     });
+
+    this.setState({
+      emptyTexture: new Texture2D(this.context.gl, {
+        data: new Uint8Array(4),
```

Comment:
is `data` needed? by default it should allocate RGBA UNSIGNED_BYTE texture, and width/height should defined  the size.